docs/reference/changelog.rst

```diff
@@ -2,6 +2,17 @@
 Changelog
 *********
 
+4.2.3 (2024-Sep-18)
+-------------------
+
+Application
+===========
+
+- get_arg_or_config now correctly checks the config service if the passed
+  namespace contains the given value but it is None.
```

Comment:
```suggestion
- ``get_arg_or_config`` now correctly checks the config service if the passed
  namespace contains the given value and the value is ``None``.
```

Reply:
The latter part of that makes it more confusing IMO. What about "if the passed namespace has ``None`` as the value of the requested item."

Reply:
Yup, that works too.